devel/ci/githubprb-project.yml

```diff
@@ -24,6 +24,7 @@
     org-list:
       - fedora-infra
     failure-comment: 'This pull request fails CI testing. Please review the Jenkins job. Hint: You can search for "JENKIES FAIL" in the console output to quickly find errors.'
+    cancel-builds-on-update: true
```

Comment:
Does that cancel current running build and then triggers a new one ?

Reply:
@cverna I assume so, but all I have to go on are these docs which do not answer that question:

https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github-pull-request

Reply:
ok :) let's try it then